ui/app/styles/core/title.scss

```diff
@@ -26,54 +22,54 @@
     color: $black;
     text-decoration: none;
   }
-  .has-font-weight-normal {
-    font-weight: $font-weight-normal;
-  }
-}
 
-.form-section .title {
-  margin-bottom: $spacing-s;
-}
+  // title sizes
+  &.is-2 {
+    font-size: $size-2;
+  }
 
-.is-subtitle-gray {
-  text-transform: uppercase;
-  font-size: $size-7;
-  color: $ui-gray-500;
-}
+  &.is-3 {
+    font-size: $size-3;
+  }
 
-// title sizes
-.title.is-2 {
-  font-size: $size-2;
-}
+  &.is-4 {
+    font-size: $size-4;
+  }
 
-.title.is-3 {
-  font-size: $size-3;
-}
+  &.is-5 {
+    font-size: $size-5;
+  }
 
-.title.is-4 {
-  font-size: $size-4;
-}
+  &.is-6 {
+    font-size: $size-6;
+  }
 
-.title.is-5 {
-  font-size: $size-5;
+  &.is-7 {
+    font-size: $size-7;
+  }
 }
 
-.title.is-6 {
-  font-size: $size-6;
+.form-section .title {
+  margin-bottom: $spacing-s;
 }
 
-.title.is-7 {
+.is-subtitle-gray {
+  text-transform: uppercase;
   font-size: $size-7;
+  color: $ui-gray-500;
 }
 
 // responsive css
 @media screen and (min-width: 769px), print {
-  .tile.is-2 {
-    flex: none;
-    width: 16.66667%;
-  }
-  .tile.is-4 {
-    flex: none;
-    width: 33.33333%;
+  .tile {
```

Comment:
Possible typo. Should this be `title`?

Reply:
Nope, it's tile. I thought the same thing and double checked. :)

Reply:
Nope, wait you're right. There is a tile class though that did mess me up. `auth-button-tile`